### PR TITLE
Adds e2e test for init command

### PR DESCRIPTION
**Pull request by seans3**
* Adds end-to-end test for `init` command. Specifically, tests the namespace calculation.
* Adds logging statement during namespace calculation

### CODE REVIEW

File: examples/alphaTestExamples/init.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+<!-- @makeWorkplace @testE2EAgainstLatestRelease -->
+```
+DEMO_HOME=$(mktemp -d)
+```
+
+Alternatively, use
+
+> ```
+> DEMO_HOME=~/hello
```
Comment (Liujingfang1) [inline examples/alphaTestExamples/init.md:17]: nit: Let's use `DEMO_HOME=~/demo`. By the `hello` folder, I thought the example was about helloworld.

Comment (seans3) [reply examples/alphaTestExamples/init.md]: Fixed.

**Pull request merged**